packages/client/hooks/useDraggableReflectionCard.tsx

```diff
@@ -229,6 +228,16 @@ const useDragAndDrop = (
         ? 'REFLECTION_GRID'
         : null
     handleDrop(atmosphere, reflectionId, drag, targetType, targetGroupId)
+    if (spotlightGroup?.id) {
+      const event = isReflectionInSpotlightResults
+        ? `Spotlight result to ${targetType === 'REFLECTION_GROUP' ? 'source' : 'result'}`
+        : `Spotlight source to ${targetType === 'REFLECTION_GROUP' ? 'result' : 'grid'}`
```

Comment:
The source can be dropped on a result or the grid. It's dropped on the grid when the source is dragged but not dropped on a result, or when there are 2+ reflections in the source group and they're ungrouped. 

Results can only be dropped on the source. If the viewer tries to drop them on another result, it will return to its original position. It's worth tracking this as we might want to rethink preventing results from being grouped onto results if users are frequently doing this.